SingularityExecutor/src/main/java/com/hubspot/singularity/executor/task/SingularityExecutorTaskCleanup.java

```diff
@@ -72,26 +72,25 @@ public TaskCleanupResult cleanup(boolean cleanupTaskAppDirectory, boolean cleanu
       return finishTaskCleanup(dockerCleanSuccess);
     }
 
-
-    if (!cleanupTaskAppDirectory) {
-      log.info("Not finishing cleanup because taskApp directory is being preserved");
-      return TaskCleanupResult.WAITING;
-    }
-
-    boolean cleanupTaskAppDirectorySuccess = cleanupTaskAppDirectory();
     boolean logTearDownSuccess = taskLogManager.teardown();
```

Comment:
This was the important line to move @baconmania . This shouldn't delete any log files, so the previous logging PR is still in effect. But it moves the task app directory cleanup to the end, so that we don't wait forever to clean logs just because we haven't cleaned taskapp yet